test/unit/layouts/default.spec.js

```diff
@@ -0,0 +1,121 @@
+import { shallowMount, createLocalVue } from '@vue/test-utils'
+import Vuex from 'vuex'
+import Default from '../../../layouts/default.vue'
+
+const localVue = createLocalVue()
+localVue.use(Vuex)
+
+localVue.component('nuxt-link', { template: '<a href="#">foo</a>' })
+localVue.component('nuxt', { template: '<div />' })
+localVue.directive('mapbox', {})
+
+describe('Default', () => {
+  let store
+  let map
+  let preferences
+
+  beforeEach(() => {
+    map = {
+      namespaced: true,
+      getters: {
+        activeDatasetsLocations: jest.fn(() => 'foo'),
+        datasetsInActiveTheme: jest.fn(() => ['bar']),
+      },
+      actions: {
+        loadPointDataForLocation: jest.fn(),
+      },
+    }
+    preferences = {
+      namespaced: true,
+      modules: {
+        theme: {
+          namespaced: true,
+          state: { active: 'dark' },
+          mutations: {
+            setActive: jest.fn(),
+          },
+        },
+      },
+    }
+    store = new Vuex.Store({
+      modules: {
+        map,
+        preferences,
+      },
+    })
+  })
+
+  test('mounts without issues', () => {
+    const wrapper = shallowMount(Default, { store, localVue })
+    expect(wrapper.vm.activeTheme).toBe(preferences.modules.theme.state.active)
+    expect(wrapper.vm.activeDatasetsLocations).toBe(
+      map.getters.activeDatasetsLocations(),
+    )
+    expect(wrapper.vm.datasetsInActiveTheme).toEqual(
+      map.getters.datasetsInActiveTheme(),
+    )
+    expect(wrapper.vm.mapboxOptions).toEqual({
+      sources: map.getters.activeDatasetsLocations(),
+      style: preferences.modules.theme.state.active,
+    })
+  })
+
+  test('dispatches loadPointDataForLocation on load-locations mapbox event', () => {
+    const wrapper = shallowMount(Default, {
```

Comment:
Test this one also with multiple locationsIds --> that it always uses only the first locationId to get the pointdata. 
Maybe even with multiple datasetIds.

Reply:
I only want to test things like how they are intended. When we actually implement support for multiple locationIds, then I want to add a test. The current scenario is sure to be out of date. Writing a test for a soon to be out of date scenario seems overhead